library/sketches-library/src/main/java/uk/gov/gchq/gaffer/sketches/function/aggregate/HyperLogLogPlusAggregator.java

```diff
@@ -42,12 +46,18 @@ public void init() {
     protected void _aggregate(final HyperLogLogPlus input) {
         if (input != null) {
             if (null == sketch) {
-                sketch = new HyperLogLogPlus(5, 5);
-            }
-            try {
-                sketch.addAll(input);
-            } catch (CardinalityMergeException exception) {
-                throw new RuntimeException("An Exception occurred when trying to aggregate the HyperLogLogPlus objects", exception);
+                try {
+                    sketch = HyperLogLogPlus.Builder.build(input.getBytes());
+                } catch (IOException e) {
+                    LOGGER.warn("Unable to clone hyper log log plus object", e);
+                    sketch = input;
+                }
+            } else {
+                try {
+                    sketch.addAll(input);
+                } catch (CardinalityMergeException exception) {
```

Comment:
Should be "final CardinalityMergeException e" to be consistent with other parts of the code.